dirsrvtests/tests/suites/ds_logs/ds_logs_test.py

```diff
@@ -1175,6 +1175,73 @@ def test_cert_personality_log_help(topology_st):
                                          r"\(currently, nsSSLPersonalitySSL attribute "
                                          r"is set to '{}'\)\..*".format(WRONG_NICK))
 
+def test_stat_index(topology_st, request):
+    """Testing nsslapd-statlog-level with indexing statistics
+
+    :id: fcabab05-f000-468c-8eb4-02ce3c39c902
+    :setup: Standalone instance
+    :steps:
+         1. Check that nsslapd-statlog-level is 0 (default)
+         2. Create 20 users with 'cn' starting with 'user_'
```

Comment:
Docstring test is complaining about "user" in this message.  Maybe it doesn't like the single quote?  Not sure, but it should be fixed

Reply:
It doesn't like `user_` because in RST it is considered a reference notation, see https://docutils.sourceforge.io/docs/user/rst/quickref.html
Underscore in this case should be escaped with backslash, i.e. `user\_`.